Nos/Models/CoreData/Author+CoreDataClass.swift

```diff
@@ -170,6 +175,7 @@ import Logger
     
     @nonobjc public class func allAuthorsRequest() -> NSFetchRequest<Author> {
         let fetchRequest = NSFetchRequest<Author>(entityName: "Author")
+        fetchRequest.sortDescriptors = [NSSortDescriptor(keyPath: \Author.hexadecimalPublicKey, ascending: false)]
```

Comment:
One of my ideas was that we should change up the sort order whenever we add creators/journalists. I'm not sure of the best way to do that, and I don't think it needs to be done here. But I just wanted to seed the thought. One dumb way would just be to sort by a different property each time. Hexadecimal key this week, display name the next, bio the next 🤷